schema_salad/main.py

```diff
@@ -129,6 +129,14 @@ def arg_parser() -> argparse.ArgumentParser:
         help="Optional copyright of the input schema.",
     ),
 
+    parser.add_argument(
+        "--codegen-parser-info",
+        metavar="parser_info",
+        type=str,
+        default=None,
+        help="Optional parser name which is accessible via resulted parser API.",
```

Comment:
```suggestion
        help="Optional parser name which is accessible via resulted parser API.",
```

Well, only available via the Python parser API for now. We should either say that, or implement it on the Java side as well.

Reply:
Done.
I fix the description of `--codegen-parser-info`.